library/Sexp/test/Main.hs

```diff
@@ -0,0 +1,18 @@
+module Main where
+
+import qualified Sexp
+import qualified Sexp.Parser
+import qualified Sexp.SimplifiedPasses
+import qualified Test.Tasty as T
+
+allCheckedTests :: T.TestTree
+allCheckedTests =
+  T.testGroup
+    "All tests that are checked"
```

Comment:
Trivial comment here, and I know this is taken from existing code, but `"All tests that are checked"` sounds odd and non-specific to me.  Would something like simply `"S-expression library tests"` read better?